src/aids/admin.py

```diff
@@ -215,7 +215,7 @@ class Media:
         EligibilityTestFilter,
         LiveAidListFilter, AuthorFilter, BackersFilter,
         PerimeterAutocompleteFilter,
-        'programs', 'categories']
+        'programs', 'categories', 'categories__theme']
```

Comment:
```suggestion
        'programs', 'categories__theme', 'categories']
```

Est-ce qu'on inverserait pas leur ordre d'affichage ? (thématiques d'abord, sous-thématiques ensuite) 😄

Reply:
Aha! On peut! :)